memacs/mu.py

```diff
@@ -66,7 +70,7 @@ def __getTimestamp(self, time, onlyDate=False):
         converts xml timestamp into org readable timestamp
         Do  6 Nov 21:22:17 2014
         """
-        time = time.strip().encode('utf-8')
+        time = time.strip()
```

Comment:
I had to remove it because it was breaking the very next line, inside the strptime(). It was complaining that it receives bytes instead of str.

I don't think the encode() call is needed here, and removing it made it work for me.